tools/fixExtensions.go

```diff
@@ -28,8 +29,6 @@ func main() {
 		fmt.Printf("\nProcessing batch %d to %d: ", batch, batch+100)
 		rows, err := db.Query(`SELECT id, raw_cert
 					FROM certificates
-					WHERE x509_certificatepolicies IS NULL OR x509_certificatepolicies='null'
-					   OR permitted_names IS NULL OR permitted_names='null'
```

Comment:
This is fun, because this query will force the script to loop on the same rows over and over. You should try running it ;)